spring-cloud-stream-module-launcher/docker-compose.yml

```diff
@@ -16,4 +17,5 @@ logsink:
   environment:
     SPRING_REDIS_HOST: redis
     MODULES: org.springframework.cloud.stream.module:log-sink:1.0.0.BUILD-SNAPSHOT
-    
\ No newline at end of file
+    SERVER_PORT: 8081
```

Comment:
do we really need to change the server port, or can we provide a mapping from 8080 (in the container)?